### PR TITLE
Add a pseudo-random number function

### PR DESCRIPTION
**Pull request by Jmallone**
The pseudorandom number function in `kernel/lib/stdlib.c`  folder.
`rand()` and `srand()`

### CODE REVIEW

File: kernel/lib/stdlib.hpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * @author Michel (JMallone) Gomes (michels@utfpr.edu.br)
+ * @brief Standard library
+ * @version 0.1
+ * @date 2021-07-20
+ *
+ * @copyright Copyright the Panix Contributors (c) 2021
+ *
+ */
+#pragma once
+#define MAX_RAND 32767
```
Comment (Kfeavel) [inline kernel/lib/stdlib.hpp]: Is this necessary to access outside of the source file? If not, I would move it out of the header.

**Review by Kfeavel — CHANGES_REQUESTED** [review]
Looks good besides that one define and needing to link to the different resources (OSDev Wiki, C Standard, etc).

File: kernel/lib/stdlib.hpp (hunk 1 above)
Comment (micahswitzer) [reply kernel/lib/stdlib.hpp]: POSIX says it should be publicly available

Comment (Kfeavel) [reply kernel/lib/stdlib.hpp]: Awesome, leave it here then. Marking as resolved.

**Comment by Jmallone** [comment]
nice, i'll change this

**Comment by Kfeavel** [comment]
@Jmallone I would also rename the files to `rand.hpp` and `rand.cpp` instead of `stdlib.*`. I know this isn't POSIX complient, but I don't think that's necessarily a goal right now. We want to (ideally) do things smarter than POSIX since we're able to start from the ground up.

**Comment by Kfeavel** [comment]
Closed since @Jmallone couldn't update the MR to point to their new branch.

**Pull request closed without merging**